cluster/ci/origin/images/release-base/log-pods.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
```
$ shellcheck cluster/ci/origin/images/release-base/log-pods.sh
 
Line 26:
        read -ra allpods <<< "$(oc get pods -l ${selector} -o jsonpath='{ range .items[*]}{ .metadata.name }{ ":" }{ .status.phase }{ " " }{ end }')"
                                               ^-- SC2086: Double quote to prevent globbing and word splitting.
 
Line 51:
                oc log ${podname}
                       ^-- SC2086: Double quote to prevent globbing and word splitting.

```

Reply:
thx, will fix

Reply:
fixed